docs/_data/components/schemas/temporary-aar-exception.yaml

```diff
@@ -0,0 +1,32 @@
+type: object
+additionalProperties: false
+description: |
+  Data structure for a temporary AAR Exception against a resource.
+  Includes a unique ID, start and (optional) end time, and a reason.  
+
+  May or may not include a set of values, depending on the business
+  context in which the temporary AAR exception is used.  
```

Comment:
What use case requires or allows for no rating to be provided?

Reply:
This came to me specifically from ISO New England.  Their expectation is that the need for this depends on the line, and why you have a temporary rating- for a communication outage, obviously you need numbers.  However, for something more physical (vegetation, equipment failure etc), their expectation is that the TO's system is still sending AARs, and that those AARs reflect the current "de-rated" condition of the line in some way.  In this scenario, the numbers don't have any meaning.  The whole point of the exception then is just for reporting and compliance purposes.

Reply:
Can we dig into this? In a recent thread the idea of a "quality flag" came up:
https://github.com/trolie/spec/issues/124#issuecomment-2248998094

If the Ratings Provider knows they are sending a non-AAR, can we add an affordance in the forecast proposal to accommodate this?

For RT ICCP, MISO has the following recommendation:
| ICCP Status | Source | State | Rating State |
|---|---|---|---|
| Good | Telemetered | Normal | Rating is DLR |
| Good | Calculated | Normal | Rating is AAR |
| Good | Estimated | Normal | Rating is recourse rating (Temporary AAR exception) |
| Good | Manual | Normal | Rating is manual override (Temporary AAR exception) |

Perhaps the forecast "Rating State" flag could simply be one of:
Ambient-adjusted
Static Resource
Manual Override

Reply:
I think it's a good idea, but it doesn't solve the problem here.  Specifically, it doesn't give you a way to set a "reason" for the exception.  The main purpose of the temporary AAR exception without a rating value is exactly that: the TO can come in and record a reason for a deviation from methodology.  

This is subtle too, as it is possible to have a temporary exception and still have a true AAR.  Granted, I think this gets a little fuzzy in terms of the FERC order's definition for temporary ratings, which is about an exception simply not being an AAR.  However, imagine the following scenario- somebody detects dangerous vegetation growth under a line, so the assumed sag is no longer appropriate.  The naïve thing to do would be to simply set a static de-rate until the vegetation is removed; perhaps a recourse rating, or even a manually computed rating that assumes another value for a safe amount of sag.  

However, it could be more sophisticated than that.  The AAR computation system could (conceivably) take the allowable line sag as a parameter, and still compute AARs, simply at a reduced capacity.  

If I were to read the FERC order like a lawyer, then maybe a temporary AAR exception isn't needed, because you're still technically getting AARs.  However, if the _intent_ is that anomalies in the ratings are transparent, then I think you have to call this an exception.

Reply:
> maybe a temporary AAR exception isn't needed, because you're still technically getting AARs

Yeah, if they continue to compute an AAR with different clearance assumptions, it's still an AAR, so no documented temporary exception is needed. Your example posits vegetation incursion, but the accelerated annealing could also be discovered in an inspection. In practice, a clearance issue is likely to result in a static derate until it is addressed either by trimming trees or changing the inputs to the ratings calculation.

If the rating is being held static but has a Ratings Obligation, it needs a temporary AAR exception. If they're able to ambiently-adjust the rating, they can continue to send forecasts. 

In RT what we hoped to use the quality flags for was like a bookmark for forced outages--"you didn't have an AAR for these intervals, here let me help you create a temp AAR exception for those intervals." A planned outage is an example of creating a temp AAR exception ahead of time. In both cases we have to require ratings.

Reply:
> A planned outage is an example of creating a temp AAR exception ahead of time. In both cases we have to require ratings.
Right, in all of MISO's cases, I think you would.  From an implementation perspective, my expectation is that we would validate that, and it will be an option in configuration.  I'm just not sure that we can force it to be required for everybody at the schema level.

Reply:
Ok, can we bookmark this for later and reach out to ISONE and SPP out-of-band? I'm just struggling to understand the use case for creating one of those without ratings.

For now, I'll merge this in the new PR #131 ok?

Reply:
Sure.